contrib/inventory/consul_io.py

```diff
@@ -133,7 +133,11 @@
 import re
 import argparse
 import sys
-import ConfigParser
+
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
```

Comment:
I would suggest using

``` python
from six.moves import configparser
```

Currently inventory scripts aren't real consistent about how they handle compat issues like configparser, but thats
closest to how the app code and modules do it (they use the six provided in module_utils, but that is not recommended for external inv scripts)

``` python
from ansible.module_utils.six.moves import configparser
```